### DISCUSSION TITLE
Android TV app issue with resume on grouped versions

### DISCUSSION THREAD

**Discussion by mattmarsden1978** [jellyfin/jellyfin#8128] [General]
If I have 2 versions of the same movie grouped together e.g. 4K and 3D versions - one of the vesions won't save the resume point. I can see it when I stop the movie and it goes to the movie info screen but when I go off that and back on it theres no resume point. If I play the other version of the movie it saves the resume point. Can this be fixed? Thanks

**Comment by sourpatched** [comment]
This would have been better as an open issue on the ATV page https://github.com/jellyfin/jellyfin-androidtv and that has been done.
